checklist_plus/editor/base.py

```
@@ -0,0 +1,1019 @@
+        project_root = os.path.dirname(os.path.dirname(__file__))
+        folder = os.path.abspath(os.path.join(project_root, "data", 'lexicons'))
+        for f in os.listdir(folder):
+            self.lexicons.update(json.load(open(os.path.join(folder, f))))
+        self.data['names'] = json.load(open(os.path.join(project_root, 'data', 'names.json')))
+        self.data['names'] = {x:set(self.data['names'][x]) for x in self.data['names']}
+        make_munch = lambda x: munch.Munch(x) if type(x) == dict else x
+        for x in self.lexicons:
+            self.lexicons[x] = [make_munch(x) for x in self.lexicons[x]]
+
+        language = get_language_code(language)
+        wikidata = pickle.load(open(os.path.join(project_root, 'data', 'wikidata.pkl'), 'rb'))
```

Ruff (0.12.2)

`pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)